wiki/Legal/Terms/en.md

```diff
@@ -80,6 +80,20 @@ osu! does not endorse any User Submission or any opinion, recommendation, or adv
 osu! does not permit copyright infringing activities and infringement of intellectual property rights on its Website, and osu! will remove all Content and User Submissions if properly notified that such Content or User Submission infringes on another's intellectual property rights.
 osu! reserves the right to remove Content and User Submissions without prior notice.
 
+## YOUR RESPONSIBILITIES IN VOLUNTEER POSITIONS
+
+osu! contains a number of volunteer teams at any given point in time that are drawn directly from the community. These teams are often invested with extra permissions and access to functions that ordinary users do not have in order to facilitate the community-driven aspect of the game. These officially supported teams are as follows:
+
+- The Beatmap Nominator Group
+- The Tournament Staff
+- The Nomination Assessment Team
+- The Global Moderation Team
+- The Support Team
```

Comment:
speaking of volunteer teams, will this "no underage" policy apply to groups such as Project Loved Team, Spotlights captains, Mentorship orgs/mentors, judges of all kinds of mapping contests, etc. given that they hold some sort of position of power as well? if that's the case, these should also be mentioned here, preferably in detail

Reply:
I assume this only applies to officially promoted community events (OWC is the big one), I think project loved might not be included in that(?) and mentorship and mapping contests are run completely independently.